cpp/README

```diff
@@ -263,6 +263,20 @@ Troubleshooting CMake via ccmake
 
 Building the library on Windows (Visual Studio)
 -----------------------------------------------
+We recommend that libphonenumber on Windows be built with Visual Studio 2015
+Update 2 (https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update2-vs)
+or later.
+
+This enables support of plain UTF-8 source files, for which you also need to specify
+the /utf-8 option in build files; see "New Options in VS2015 Update 2" at
+https://blogs.msdn.microsoft.com/vcblog/2016/02/22/new-options-for-managing-character-sets-in-the-microsoft-cc-compiler/,
+or a Chrome patch for an example:
+https://codereview.chromium.org/2488853002/diff/80001/build/config/compiler/BUILD.gn
+
+Without this version and the option, your compilation will likely break when
```

Comment:
I don't know about a policy. I was paraphrasing best as I could from jungshik's explanations, I still don't have VS so don't appreciate the nuances :) I updated the language, WDYT?